### ISSUE TITLE
Dark Theme?

### ISSUE THREAD

**Issue by amancevice** [apache/superset#4040]
It would be nice if superset supported a 'dark' theme out of the box so that the app and its dashboards can easily be dimmed without writing a very extensive custom CSS.

**Comment by visshaljagtap** [comment]
Yes it will be helpful and also it would be great if we have custom theme option.

**Comment by agabardo** [comment]
Any news on this?

**Comment by visakhmurukes** [comment]
Any updates on this?
Would be a great addition to have this

**Comment by leon-do** [comment]
bump?

**Comment by visakhmurukes** [comment]
I ended up adding a custom CSS for the dashboard.
All except table looks fine at the moment

**Comment by DrissiReda** [comment]
@visakhmurukes could you please share that custom CSS

**Comment by visakhmurukes** [comment]
@DrissiReda  here you go



@import 'https://fonts.googleapis.com/css?family=Open+Sans';

* {
	-webkit-box-sizing: border-box;
	box-sizing: border-box;
}

body{
	font-family: 'Open Sans', sans-serif;
	line-height: 1.75em;
	font-size: 16px;
	background-color: #222;
	color: #aaa;
}
.superset-legacy-chart-table {
    background-color: #333 !important;
}

.nvtooltip table td, .nvtooltip table td strong.x-value {
color:#222;
}

iframe {
width: 80%;
height: 80%;
}

.table{
   background-color:#333 !important;
}

.table .table{
   background-color:#333 !important;
}

.table-striped>tbody>tr:nth-of-type(odd){
background-color:#333 !important;
}

.dashboard-component-chart-holder {
   background-color: #303030;
   border-radius: 5px;
}
.header-line {
  color: #f5f5f5;
}

.nv-legend-text{
 fill:#fcfcfc !important;
}
.treemap text tspan{
   fill: midnightblue;
   text-shadow:none;
}

.nv-label text {
 fill:#fcfcfc !important;

}
.nvd3.nv-pie.nv-slice path{
stroke:#c5c5c5;
}

text {
 fill: #fcfcfc;
text-shadow:0 0 1px #000;
}

.dashboard-component-chart-holder{
color:#c2c2c2;
}

.simple-container {
	max-width: 675px;
	margin: 0 auto;
	padding-top: 70px;
	padding-bottom: 20px;
}

.Select-control{
background-color: #ccc;
}
.Select-value-label{
color: #c66e01e0;
}

.simple-print {
	fill: white;
	stroke: white;
}
.simple-print svg {
	height: 100%;
}

.simple-close {
	color: white;
	border-color: white;
}

.simple-ext-info {
	border-top: 1px solid #aaa;
}

p {
	font-size: 16px;
}

h1 {
	font-size: 30px;
	line-height: 34px;
}

h2 {
	font-size: 20px;
	line-height: 25px;
}

h3 {
	font-size: 16px;
	line-height: 27px;
	padding-top: 15px;
	padding-bottom: 15px;
	border-bottom: 1px solid #D8D8D8;
	border-top: 1px solid #D8D8D8;
}

hr {
	height: 1px;
	background-color: #d8d8d8;
	border: none;
	width: 100%;
	margin: 0px;
}

a[href] {
	color: #1e8ad6;
}

a[href]:hover {
	color: #3ba0e6;
}

img {
	max-width: 100%;
}

li {
	line-height: 1.5em;
}

aside,
[class *= "sidebar"],
[id *= "sidebar"] {
	max-width: 90%;
	margin: 0 auto;
	border: 1px solid lightgrey;
	padding: 5px 15px;
}

@media (min-width: 1921px) {
	body {
		font-size: 18px;
	}
}

**Comment by DrissiReda** [comment]
Thanks! any way that can be added to the whole superset ui?

**Comment by eugeniamz** [comment]
@junlincc  FYI

**Comment by scimas** [comment]
bump

**Comment by junlincc** [comment]
Reopening and it's pinned.

**Comment by suddjian** [comment]
This is definitely being worked towards as Superset's component styles are switched to Emotion, but it will be quite a bit of work to get there. Relevant: #9123

**Comment by avaranze** [comment]
Any update on this since #9123 is merged?
